packages/cloud/package.json

```
@@ -39,8 +39,7 @@
     "dependencies": {
         "@pankod/refine-core": "^3.25.0",
         "@pankod/refine-sdk": "^0.0.25",
-        "react-hook-form": "^7.30.0",
-        "react-query": "^3.19.2"
+        "lodash.merge": "^4.6.2"
```

```suggestion
        "lodash.merge": "^4.6.2"
```
we have to manage the lodash dependency as in core

https://github.com/pankod/refine/blob/next/packages/core/package.json#L43
https://github.com/pankod/refine/blob/next/packages/core/package.json#L44

https://github.com/pankod/refine/blob/next/packages/core/tsup.config.ts#L16